corehq/apps/domain/calculations.py

```diff
@@ -74,7 +75,9 @@ def active_mobile_users(domain, *args):
     return num_users if 'inactive' not in args else len(user_ids) - num_users
 
 
-def cases(domain, *args):
+def cases(domain, primary_db=False, *args):
```

Comment:
These two methods aren't used right now (they were earlier in the PR). keeping htem here becasue I think they will fit in well for proactive alerting (to hopefully come soon)